dotnet/tests/Microsoft.Agents.AI.Workflows.UnitTests/AgentWorkflowBuilderTests.cs

```diff
@@ -159,7 +159,7 @@ public override async IAsyncEnumerable<AgentRunResponseUpdate> RunStreamingAsync
 
     private sealed class DoubleEchoAgentThread() : InMemoryAgentThread();
 
-    [Fact]
+    [Fact(Skip = "issue #1109")]
```

Comment:
This is fine, but note that the problem isn't in the test but rather in the workflow implementation. Until those threading issues are fixed, _any_ workflow that might have nodes running concurrently will be susceptible to such random failures.